### PR TITLE
delete head branch of mergify backport

### PR DESCRIPTION
**Pull request by sileht**

### CODE REVIEW

File: mergify_engine/tasks/engine/v2.py [hunk 1]
```diff
@@ -141,6 +141,13 @@ def run_actions(installation_id, installation_token, subscription,
 def handle(installation_id, installation_token, subscription,
            pull_request_rules_raw, event_type, data, pull_raw):
 
+    # Some mandatory rules
```
Comment (jd) [inline mergify_engine/tasks/engine/v2.py]: It makes the engine depends on a plugin (`delete_head_branch`).

File: mergify_engine/tasks/engine/v2.py [hunk 2]
```diff
@@ -141,6 +141,13 @@ def run_actions(installation_id, installation_token, subscription,
...
+    pull_request_rules_raw["rules"].extend([
+        {"name": "delete backport branch (Mergify rule)",
+         "conditions": ["head~=^mergify/bp/", "closed"],
```
Comment (jd) [inline mergify_engine/tasks/engine/v2.py]: Maybe add also author?
Not sure we have a config variable available with that author name available though.

**Comment by sileht** [comment]
Pull-request updated, HEAD is now 80216ca5820f53a8a0f024abd78201dd6d3a319f

File: mergify_engine/tasks/engine/v2.py (hunk 1 above)
Comment (sileht) [reply mergify_engine/tasks/engine/v2.py]: I have moved this into a configuration file.

**Comment by sileht** [comment]
If I move that into action backport, I must add a hook that exec on every evaluation.

**Comment by sileht** [comment]
Pull-request updated, HEAD is now d574eb481b16da08209e88816cc79b416fbebc0f

**Pull request merged**